tools/update_snapcraft.py

```diff
@@ -157,14 +167,15 @@ def clone_releases_repository(reuse):
 def main(args):
     """Entry point to the application."""
     clone_releases_repository(args.reuse)
+    excluded_plugins = EXCLUDED_PLUGINS_PATH.read_text().split("\n")
```

Comment:
I think that comments will not cause an issue, and it's file to keep them there.

Reply:
yeah, since the match is on the entire line, comment lines don't affect the output; on the other hand, in-line comments are not handled. I was just being hasty :innocent: